### PR TITLE
Feature/us8779 new famiy add address

### PR DESCRIPTION
**Pull request by timmyg**
build: https://goo.gl/C3f8SF

### CODE REVIEW

File: SignInCheckIn/MinistryPlatform.Translation/Models/DTO/MpAddressDto.cs [hunk 1]
```diff
@@ -18,7 +18,7 @@ public class MpAddressDto
         [JsonProperty(PropertyName = "City")]
         public string City { get; set; }
 
-        [JsonProperty(PropertyName = "State")]
+        [JsonProperty(PropertyName = "State/Region")]
```
Comment (jpc11623) [inline SignInCheckIn/MinistryPlatform.Translation/Models/DTO/MpAddressDto.cs]: You might want to escape this as [State/Region]. SQL/MP will blow up on something with a hyphen in the name, not sure of side effects with this.

File: SignInCheckIn/MinistryPlatform.Translation/Repositories/ParticipantRepository.cs [hunk 2]
```diff
@@ -305,12 +305,12 @@ public MpHouseholdDto GetHouseholdByHouseholdId(string token, int householdID)
 
         public void UpdateHouseholdInformation(string token, MpHouseholdDto householdDto)
         {
-            var columns = new List<string>
+            var householdIdColumn = new List<string>
```
Comment (jpc11623) [inline SignInCheckIn/MinistryPlatform.Translation/Repositories/ParticipantRepository.cs]: If it's a list, probably should be named as a plural.

File: SignInCheckIn/MinistryPlatform.Translation/Models/DTO/MpAddressDto.cs (hunk 1 above)
Comment (timmyg) [inline SignInCheckIn/MinistryPlatform.Translation/Models/DTO/MpAddressDto.cs]: Jim mentioned that - but we looked at crds-angular and it was unescaped. So you think we should still put brackets around it? Your call as Im not as experienced

File: SignInCheckIn/MinistryPlatform.Translation/Repositories/ParticipantRepository.cs (hunk 2 above)
Comment (timmyg) [inline SignInCheckIn/MinistryPlatform.Translation/Repositories/ParticipantRepository.cs]: true true

File: SignInCheckIn/MinistryPlatform.Translation/Models/DTO/MpAddressDto.cs (hunk 1 above)
Comment (jpc11623) [inline SignInCheckIn/MinistryPlatform.Translation/Models/DTO/MpAddressDto.cs]: Yeah, I think it should be escaped.

**Pull request merged**